data/abilities/lateral-movement/aa6ec4dd-db09-4925-b9b9-43adeb154686.yml

```diff
@@ -11,7 +11,7 @@
     windows:
       psh:
         command: |
-          net use \\#{remote.host.fqdn}\C$ /user:#{network.domain.name}\#{domain.user.name} #{domain.user.password}
+          net use \\#{remote.host.fqdn}\C$ /user:#{domain.user.name} #{domain.user.password}
```

Comment:
Why was the `network.domain.name` part removed? What if the user exists as both a local user and a domain user, or the domain of the agent's machine doesn't match the one it's trying to laterally move to?

Reply:
It's a bug fix. At some point (not sure exactly when) the network.domain.name's value started getting included in the domain.user.name fact. I suspect this may have been a change in the katz parser at some point. I considered changing it back, but several abilities have already been swapped over to remove the network.domain.name fact, and it made more sense to do less overhaul. 

i.e. before: `/user:caldera/caldera/admin` -> after: `/user:caldera/admin`

Reply:
Because the domain information seems to be encoded in the username field, I think we're safe in the situation of a local user/domain user collision, but it's hard to know for sure.